components/console-backend-service/hack/update-mocks.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+mockery -case underscore -name DiscoveryInterface -dir ./vendor/k8s.io/client-go/discovery/ -output ./internal/domain/k8s/automock -outpkg automock
```

Comment:
Any chance we could make it a `go generate` directive instead of additional script?

Reply:
I could add 
```
//go:generate mockery -name=DiscoveryInterface -case=underscore -output=automock -outpkg=automock -dir ../../../vendor/k8s.io/client-go/discovery/
```
to resource_service.go
But I do not like the relative path with ../../../ cause it will fail when this file moves in the directory structure.

Reply:
How about adding a simple wrapper interface? 
```
type DiscoveryInterface interface {
  discovery.DiscoveryInterface
}
```
and mock this one? Then IDE will update paths whenever you move the file.

Reply:
Right, fixed